requirements.txt

```diff
@@ -1,31 +1,25 @@
-# Optionals
-graphene; python_version<'3.10'
-itsdangerous
-jinja2
-python-multipart
-pyyaml
-requests
+-e .[full]
```

Comment:
Do we really want to do `[full]`? These dependencies then won't be pinned and so will only be updated implicitly when the cache is invalidated.